backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/lower/InteropLowering.kt

```diff
@@ -839,11 +834,11 @@ private class InteropTransformer(val context: Context, override val irFile: IrFi
 
         expression.transformChildrenVoid(this)
         builder.at(expression)
-        val descriptor = expression.descriptor.original
         val function = expression.symbol.owner
 
-        if (descriptor == interop.nativePointedRawPtrGetter ||
-                OverridingUtil.overrides(descriptor, interop.nativePointedRawPtrGetter, false)) {
+        // TODO: this is fragile.
```

Comment:
Yes it is.
What I meant is to check here that class derives from `NativePointed` and that this function is getter of property named `rawPtr`.
In this case this property overrides `NativePointed.rawPtr`.

Btw, it is possible that overriding check is not required anymore, after this patch: https://github.com/JetBrains/kotlin-native/commit/2b00850164d6aa5680248ac9ed7f3d35ec5ef1f9#diff-87031cea584971a666c72def5450f827R27